fudge-ratio/java/servlets/FudgeRatio.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
```

Comment:
Are we going to use this?

Reply:
No, that should have been removed after Oscar's comment.